judge/views/api/api_v2.py

```diff
@@ -236,7 +236,7 @@ def get_object_data(self, contest):
             'has_rating': contest.ratings.exists(),
             'rating_floor': contest.rating_floor,
             'rating_ceiling': contest.rating_ceiling,
-            'hidden_scoreboard': contest.hide_scoreboard,
```

Comment:
Do we want to keep this in for backwards compatibility? (`hidden_scoreboard': contest.scoreboard_visibility in ('Contest', 'Participation')`)

/cc @Xyene @quantum5